src/models/utils/selectablefilter.ts

```diff
@@ -0,0 +1,6 @@
+import { Filter } from '@yext/answers-core';
+
+export interface SelectableFilter {
```

Comment:
Can this extend Filter rather than have a filter as a prop? That way it would be more similar to DisplayableFacet which extends Facet. Also, would the name `StaticFilter` make sense?

Reply:
I think internally it would be cleaner/easier to keep filter as its own object in `SelectableFilter ` instead of spreading it out into multiple fields so when we don't have to remake a filter type when transforming it to core's nested filter structure and whatnots. But I also don't have a strong preference on this, so it can be change if you think it's still better!
As for the name, I think we should still keep it as `SelectableFilter` since it's not strictly specific to static, just that it holds a selected option (maybe can reuse for some other filters, likeFilterSearch?). `StaticFilter` name might be confusing since it's so similar to `StaticFilters` component

Reply:
Creating a simple public interface is more important than how simple the code is internally. As to what's the simpler public interface, I am leaning towards having `SelectableFilter` extend `Filter` because that way all the fields can be on one documentation page of what a filter is. However I think we should talk to product because it's ultimately up to them

Reply:
and keeping `SelectableFilter` sgtm

Reply:
Update: Product is fine with filter as a field in SelectableFilter. Keeping `SelectableFilter` as the name

Reply:
I'm okay with leaving this interface as-is, however it's worth noting that even if this were updated so that it extends Filter, we'd wouldn't need to remake a filter type when transforming it to core's nested filter structure because SelectableFilter is still a Filter. We could pass in SelectableFilters to core as Filters and it should work without needing any additional transformations. The `selected` field wouldn't do anything which is a potential point of confusion, but typescript would be okay with it because of duck typing.